### PR TITLE
Fileshare refactor to devtools

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: sdk/storage/azure-storage-file-share/tests/test_file.py [hunk 1]
```diff
@@ -46,53 +46,54 @@
...
-                pass
-
-        return super(StorageFileTest, self).tearDown()
+        remote_url = self.get_file_url(rmt_account_name)
+        remote_credential = rmt_key
+        if remote_url:
+            self.fsc2 = ShareServiceClient(remote_url, credential=remote_credential)
+            self.remote_share_name = None
+
+    # def tearDown(self):
```
Comment (kristapratico) [inline sdk/storage/azure-storage-file-share/tests/test_file.py]: just remove this?

File: sdk/storage/azure-storage-file-share/tests/test_file_async.py [hunk 2]
```diff
@@ -59,93 +60,88 @@ class AiohttpTestTransport(AioHttpTransport):
...
-                pass
-
-        return super(StorageFileAsyncTest, self).tearDown()
+        remote_url = self.get_file_url(rmt_account_name)
+        remote_credential = rmt_key
+        if remote_url:
+            self.fsc2 = ShareServiceClient(remote_url, credential=remote_credential, transport=AiohttpTestTransport())
+            self.remote_share_name = None
+
+    # def tearDown(self):
```
Comment (kristapratico) [inline sdk/storage/azure-storage-file-share/tests/test_file_async.py]: same here?

File: sdk/storage/azure-storage-file-share/tests/test_get_file.py [hunk 3]
```diff
@@ -61,27 +60,27 @@ def setUp(self):
...
             )
             file_client.upload_file(self.byte_data)
 
-    def tearDown(self):
-        if not self.is_playback():
-            try:
-                self.fsc.delete_share(self.share_name, delete_snapshots='include')
-            except:
-                pass
+    # def tearDown(self):
```
Comment (kristapratico) [inline sdk/storage/azure-storage-file-share/tests/test_get_file.py]: remove comments here and below

File: sdk/storage/azure-storage-file-share/tests/test_get_file_async.py [hunk 4]
```diff
@@ -42,18 +42,34 @@ class AiohttpTestTransport(AioHttpTransport):
         return response
 
 
-class StorageGetFileTest(FileTestCase):
-    def setUp(self):
-        super(StorageGetFileTest, self).setUp()
+class StorageGetFileTest(AsyncStorageTestCase):
+    # def tearDown(self):
```
Comment (kristapratico) [inline sdk/storage/azure-storage-file-share/tests/test_get_file_async.py]: more commented out code

File: sdk/storage/azure-storage-file-share/tests/test_share_async.py [hunk 5]
```diff
@@ -55,28 +55,30 @@ class AiohttpTestTransport(AioHttpTransport):
...
+        credentials = storage_account_key
         self.fsc = ShareServiceClient(account_url=file_url, credential=credentials, transport=AiohttpTestTransport())
-        loop = asyncio.get_event_loop()
-        loop.run_until_complete(self.fsc.__aenter__())
         self.test_shares = []
 
-    def tearDown(self):
-        if not self.is_playback():
-            loop = asyncio.get_event_loop()
+    # def tearDown(self):
```
Comment (kristapratico) [inline sdk/storage/azure-storage-file-share/tests/test_share_async.py]: another tearDown

**Comment by xiafu-msft** [comment]
😊 Please don't merge it until the devtool refactor pr get merged  https://github.com/Azure/azure-sdk-for-python/pull/8930
It was tough when new features which requires preprod account or stage account were added.

**Comment by rakshith91** [comment]
> Please don't merge it until the devtool refactor pr get merged #8930
> It was tough when new features which requires preprod account or stage account were added.

That refactoring needs to be done anyway..ill do it together along with blobs and queues...want to prioritize this

File: sdk/storage/azure-storage-file-share/tests/test_handle.py [hunk 6]
```diff
@@ -32,11 +33,12 @@
 
 # ------------------------------------------------------------------------------
 
+@pytest.mark.skip("Need to setup handles again")
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-file-share/tests/test_handle.py]: the whole file is skipped

File: sdk/storage/azure-storage-file-share/tests/test_get_file_async.py [hunk 7]
```diff
@@ -41,19 +41,19 @@ class AiohttpTestTransport(AioHttpTransport):
             response.content_type = response.headers.get("content-type")
         return response
 
+@pytest.mark.live_test_only
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-file-share/tests/test_get_file_async.py]: in the old test framework, it's unnecessary to run every test in live mode.

**Pull request merged**